File: java/Thread/p1.java
import java.util.*;

class MyThread implements Runnable{
	public void run(){
		System.out.println("inside thread"+ Thread.currentThread());

	}

}


class CustomThread extends Thread{
	CustomThread(String name){

		super(name);
	}
	
	public void run(){

		System.out.println("inside thread"+ Thread.currentThread());
	}

}

class Test{
	public static void main(String... args){
		Thread obj = new Thread(new MyThread());
		obj.start();


		Thread t = new CustomThread("Custom Thread name");
		t.start();

}
}

File: CPP/inheritence/p1.cpp
#include<iostream>
using namespace std;

class A{


	public: 
		A(){
			cout<<"const A"<<endl;
	
		}


};


class B{


	public: 
		B(){
			cout<<"const B"<<endl;
	
		}


};


class C:public B,public A{


	public: 
		C(){
			cout<<"const C"<<endl;
	
		}


};





int main(){
	C obj;
	
	



}
